Code/Framework/AzCore/AzCore/Math/Internal/SimdMathVec4_scalar.inl

```diff
@@ -945,6 +945,21 @@ namespace AZ
         }
 
 
+        AZ_MATH_INLINE void Vec4::Mat4x4MultiplyAdd(const FloatType* rowsA, const FloatType* rowsB, FloatType* out)
+        {
+            for (int32_t row = 0; row < 4; ++row)
+            {
+                for (int32_t col = 0; col < 4; col++)
+                {
+                    for (int32_t k = 0; k < 4; ++k)
+                    {
+                        out[row].v[col] += rowsA[row].v[k] * rowsB[k].v[col];
+                    }
+                }
+            }
+        }
+
+
```

Comment:
Same as above, it would be more intuitive to me if this had a rowsC input parameter that's separate from the output.